frontend/src/scenes/data-warehouse/editor/sidebar/QueryInfo.tsx

```diff
@@ -151,6 +155,34 @@ export function QueryInfo({ codeEditorKey }: QueryInfoProps): JSX.Element {
                                         loading={updatingDataWarehouseSavedQuery}
                                         options={OPTIONS}
                                     />
+                                    <LemonButton
+                                        type="secondary"
+                                        size="small"
+                                        tooltip="Revert materialized view to view"
+                                        disabledReason={
+                                            savedQuery?.status === 'Running' &&
+                                            'Cannot revert while materialization is running'
+                                        }
+                                        icon={<IconX />}
```

Comment:
Could we pick a better icon? Either a trash icon or better yet, we have a `<IconRevert />` icon!